flac.go

```diff
@@ -403,20 +374,17 @@ func (stream *Stream) makeSeekTable() (err error) {
 	var sampleNum uint64
 	var tmp []meta.SeekPoint
 	for {
+		o, err := rs.Seek(0, io.SeekCurrent)
```

Comment:
```suggestion
		// Record seek offset to start of frame.
		o, err := rs.Seek(0, io.SeekCurrent)
```

Also, consider renaming `o` to e.g. `off`.